### PR TITLE
Add documents page

### PR DESCRIPTION
**Pull request by sarunast**
Screens:
screen shot 2018-03-27 at 11 56 38
screen shot 2018-03-27 at 12 11 47

The fonts might be repeating I will have a look if I can reuse them.

### CODE REVIEW

**Comment by sarunast** [comment]
Deploy preview for *masifunde* ready!

Built with commit 2e268a0ab025fcad17de508ca761788deb4d7662

https://deploy-preview-215--masifunde.netlify.com

**Comment by erikjalevik** [comment]
Shouldn't there be a link to the Documents page in the footer? Under Blog?

**Comment by erikjalevik** [comment]
Looks good in all widths!

File: api/common.js [hunk 1]
```diff
@@ -13,10 +13,37 @@ export async function fetchFooterData(locale) {
...
+      ...unwrappedFields,
+      file: unwrapFile(unwrappedFields.file),
+    }
+  })
+
 // For a list of valid image URL parameters, see the Contentful API docs:
 // https://www.contentful.com/developers/docs/references/images-api
 export const unwrapImage = (image, urlParams) => {
-  const imageFile = image && image.fields && image.fields.file
+  const imageFile = unwrapFile(image)
```
Comment (erikjalevik) [inline api/common.js]: 👍

**Comment by sarunast** [comment]
Added the link to the footer and changed the `teamMember` component to be with text.

File: pages/documents/index.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  teamMember,
+  bannerButtonText,
+  bannerButtonUrl,
+}) => (
+  <Fragment>
+    <Head title={metaTitle} description={metaDescription} />
+
+    <PageSection>
+      <h1>Test</h1>
+      <CenteredText source="centered Text" />
```
Comment (erikjalevik) [inline pages/documents/index.js]: This need to come from Contentful, doesn't it?

**Pull request merged**